src/content/docs/logs/get-started/logging-best-practices.mdx

```diff
@@ -0,0 +1,158 @@
+ />
+
+* Use external FluentBit configurations for parsing multi-line logs and for other, more extensive pre-parsing before ingesting into New Relic. For details and configuration of multi-line parsing with our infrastructure agent, see [this blog post](https://newrelic.com/blog/how-to-relic/parse-multiline-log-messages-fluent-bit-plugin).
+
+* Create optimized Grok patterns to match the filtered logs to extract attributes.  Avoid using expensive Grok patterns like GREEDYDATA excessively. Ask your account team or SC to help identify any sub-optimal parsing rules.
+
+**GROK Best Practices:**
+ * Use Grok types to specify the type of attribute value to extract. If omitted, values are extracted as strings. This is important especially for numerical values if you want to be able to use NRQL functions (i.e., `monthOf()`, `max()`, `avg()`, &`lt;`, `&gt;`, etc.) on these attributes.
+* Use the **Parsing** UI to test your Grok patterns. You can paste sample logs in the **Parsing** UI to validate your Grok or Regex patterns and if they're extracting the attributes as you expect.
+ * Add anchors to parsing logic (i.e., `^`) to indicate the beginning of a line, or `$` at the end of a line.
+ * Use `()?` around a pattern to identify optional fields
+ * Avoid overusing expensive Grok patterns like `'%{GREEDYDATA}`. Try to always use a valid Grok pattern and Grok type when extracting attributes.
+
+## Drop Filter Rules [#drop-rules]
+
+### Drop logs at ingest
+
+* Create [drop filter rules](/docs/logs/ui-data/drop-data-drop-filter-rules#create) to drop logs that are not useful or that are not required to satisfy any use cases for dashboards, alerts or troubleshooting
+
+### Drop attributes from your logs at ingest
+
+* Create drop rules to drop unused attributes from your logs.
+* Drop the `message` attribute after parsing. If you parse the message attribute to create new attributes from the data, drop the message field.
+* Example: If you're forwarding data from AWS infrastructure, you can create drop rules to drop any AWS attributes that may create unwanted data bloat.
+
+## New Relic Logs Sizing [#sizing]
+
+* How we bill for storage may differ from some of our competitors. How we meter for log data is similar to how we meter and bill for other types of data, which is defined in [Usage calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#usage-calculation).
+* If our cloud integrations (AWS, Azure, GCP) are installed, we will add cloud metadata to every log record, which will add to the overall ingest bill. This data can be dropped though to reduce ingest. A public screen recording showing how you can drop the AWS metadata can be found [here](https://drive.google.com/file/d/1bqEu2b6CxX3jfcp0ggJDj_2ckquGc7aP/view?usp=sharing).
+* The main drivers for log data overhead are below, in order of impact:
+ * Cloud integrations
+ * JSON formatting
+ * Log patterns (You can disable/enable patterns in the **Logs** UI.)
```

Comment:
Where in the Logs UI can you do this? I'll add quick instructions or a docs link here.